lib/Watching.js

```diff
@@ -126,12 +125,9 @@ class Watching {
 		this.compiler.removedFiles = this._collectedRemovedFiles;
 		this._collectedRemovedFiles = undefined;
 
-		this.compiler.fileTimestamps =
+		this.compiler.fileTimestamps = this.compiler.contextTimestamps =
 			fileTimeInfoEntries ||
 			(this.pausedWatcher && this.pausedWatcher.getFileTimeInfoEntries());
-		this.compiler.contextTimestamps =
```

Comment:
this is where, before, contextTimestamps and fileTimestamps are both created as separate copies of eachother.